platform/security/pep/security-pep-interceptor/src/main/java/ddf/security/pep/interceptor/PEPAuthorizingInterceptor.java

```diff
@@ -118,39 +114,30 @@ public void handleMessage(Message message) throws Fault {
                     isPermitted = user.isPermitted(action);
 
                     logger.debug("Result of permission: {}", isPermitted);
-                    SecurityLogger.logInfo(
-                            "Is user [" + user.getPrincipal() + "] permitted: " + isPermitted);
+                    SecurityLogger.audit("Is Subject  permitted? " + isPermitted, user);
                     // store the subject so the DDF framework can use it later
                     ThreadContext.bind(user);
                     message.put(SecurityConstants.SAML_ASSERTION, user);
                     logger.debug("Added assertion information to message at key {}",
                             SecurityConstants.SAML_ASSERTION);
                 } catch (SecurityServiceException e) {
-                    logger.warn("Caught exception when trying to perform AuthZ.", e);
-                    SecurityLogger.logWarn(
+                    SecurityLogger.audit(
```

Comment:
I don't think so. I think of auditWarn more as "I'm about to elevate privileges" or "I'm about to do something kind of stupid". I don't think that audit and auditWarn map to info and warn quite like they do in the normal logging sense where you might elevate to warn if there is an exception. The opposite is also true.